### DISCUSSION TITLE
How to stay one minor "behind", but keep patches coming?

### DISCUSSION THREAD

**Discussion by illrill** [renovatebot/renovate#25970] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### What is your question?

How can I make Renovate always stay one minor behind, but still keep patches coming?

Let's say a certain dependency has the following versions:

1.11.0
1.12.0

I want Renovate to provide patches for 1.11.* but ignore anything that has to do with 1.12 until the dependency has released a 1.13. As soon as there is a 1.13 available, I want Renovate to suggest that I upgrade to the latest patch on 1.12.

**Comment by rarkins** [comment, marked as answer]
You can specify separateMinorPatch=true to get both patch and minor updates if both are available. You cannot "stay one minor release behind" though

**Reply by illrill** [reply]
Thanks for the reply. One idea I considered was to try and somehow capture and feed my `currentVersion` into the `packageRules.allowedVersions` regex matcher to achieve this. Based on your experience, could that work? Or can you think of any other solution I can explore?

**Reply by rarkins** [reply]
It might be possible to achieve that with allowedVersions and some templating based on currentVersion, but that might need an enhancement too
